log/log.go

```
@@ -70,17 +70,28 @@ func (aw *asyncWriter) process() {
 	defer aw.wg.Done()
 	for {
 		select {
-		case entry := <-aw.logChan:
+		case entryBufPtr := <-aw.logChan: // entryBufPtr is *[]byte
 			aw.logger.outMu.Lock()
-			aw.logger.out.Write(entry)
+			aw.logger.out.Write(*entryBufPtr)
```

golangci-lint (1.64.8)

Error return value of `aw.logger.out.Write` is not checked

(errcheck)